misc/chip_data.json

```diff
@@ -7355,7 +7355,7 @@
     },
     "226fafe2-1ea8-4ae8-861e-ae4b1bd10bfe": {
       "ReadonlyName": "Substring",
-      "Description": "",
+      "Description": "Spits string values",
```

Comment:
I think we can go more in-depth here. Maybe something like "Allows you to extract individual characters from a string, you can think of this like a list of characters"